sp800-63b/sec4_aal.md

```diff
@@ -50,11 +50,11 @@ The CSP SHOULD employ appropriately-tailored security controls from the low base
 
 #### <a name="aal1records"></a> 4.1.5. Records Retention Policy
 
-The CSP shall comply with its respective records retention policies in accordance with applicable laws, regulations, and policies, including any National Archives and Records Administration (NARA) records retention schedules that may apply. If the CSP opts to retain records in the absence of any mandatory requirements, the CSP SHALL conduct a risk management process, including asessements of privacy and security risks to determine how long records should be retained and SHALL inform the subscriber of that retention policy.
+The CSP shall comply with its respective records retention policies in accordance with applicable laws, regulations, and policies, including any National Archives and Records Administration (NARA) records retention schedules that may apply. If the CSP opts to retain records in the absence of any mandatory requirements, the CSP SHALL conduct a risk management process, including assessements of privacy and security risks to determine how long records should be retained and SHALL inform the subscriber of that retention policy.
 
 ### 4.2. Authenticator Assurance Level 2
 
-AAL2 provides high confidence that the claimant controls authenticator(s) bound to the subscriber's account. Proof of possession and control of two different authentication factors is required through a secure authentication protocol. In order to authenticate at AAL2, claimants SHALL successfully authenticate using both authentication factors. Approved cryptographic techniques are required at AAL2 and above.
+AAL2 provides high confidence that the claimant controls authenticator(s) bound to the subscriber's account. Proof of possession and control of authenticators demonstrating two different authentication factors out of the three (*something you know*, *something you have*, and *something you are*) is required. In order to authenticate at AAL2, claimants SHALL successfully authenticate using both authentication factors. Secure authentication protocols and approved cryptographic techniques SHALL be used.
```

Comment:
"Distinct" it is. There is still a fair amount of inconsistency between the AAL introductions in 4.x and those in the -3 Executive Summary, but I'm not making the -3 changes here because the version in this branch is substantially different from what is in the main branch now.